go/arrow/compute/internal/exec/span.go

```diff
@@ -590,7 +590,7 @@ type ExecSpan struct {
 
 func getNumBuffers(dt arrow.DataType) int {
 	switch dt.ID() {
-	case arrow.NULL, arrow.STRUCT, arrow.FIXED_SIZE_LIST:
+	case arrow.NULL, arrow.STRUCT, arrow.FIXED_SIZE_LIST, arrow.RUN_END_ENCODED:
```

Comment:
I thought REE arrays had no buffers, not even a validity buffer?

Reply:
good point